ui/src/app/guards/workspace-guard.service.ts

```diff
@@ -13,7 +14,8 @@ export class WorkspaceGuard implements CanActivate, CanActivateChild {
     private router: Router) {}
 
   canActivate(route: ActivatedRouteSnapshot, state: RouterStateSnapshot): Observable<boolean> {
-    if (route.routeConfig.path === 'data' ||
+    console.log(serverConfigStore.getValue().enableResearchReviewPrompt);
+    if (serverConfigStore.getValue().enableResearchReviewPrompt && route.routeConfig.path === 'data' ||
```

Comment:
Is this the right if statement? Seems like this should be further down @ line 21

Reply:
Since enableResearchReviewPrompt is false there is no point even to have the api call this.workspaceService.getWorkspace to check for accessLevel or anything and just return 
Observable.from([true])

Reply:
I see - I didn't realize this was a new guard added only for this new review prompt feature. Lets discuss more, I have performance concerns with this approach and I think there may be some inconsistent behavior, but it's all feature flagged with this PR so it's not urgent.